### PR TITLE
MINT-1762 Eliminate core dependencies in ApprovalNoticeModel

### PR DESCRIPTION
**Pull request by olegdizus**

### CODE REVIEW

File: src/components/common/Snackbar/Snackbar.jsx [hunk 1]
```diff
@@ -62,10 +63,16 @@ export default class Snackbar extends PureComponent {
     return (
       <div styleName={classnames('snackbar', { 'error-notice': isErrorNotice })}>
         <span styleName='snackbar-entry'>
-          <span styleName='entry-status'>{notice.title()}</span>
+          <Translate
```
Comment (ozalexo) [inline src/components/common/Snackbar/Snackbar.jsx]: Looks like it should be 
```
<span styleName='entry-status'>
  <Translate
    styleName='entry-status'
    value={notice.title()}
  />
</span>

File: packages/core/models/notices/WalletNoticeModel.js [hunk 2]
```diff
@@ -44,8 +42,11 @@ export default class WalletNoticeModel extends abstractNoticeModel({
   }
 
   message () {
-    return I18n.t(this.get('action'), {
-      address: this.get('address'),
-    })
+    return [
```
Comment (ozalexo) [inline packages/core/models/notices/WalletNoticeModel.js]: In some place message returns {value: ''}, but here we have an array... Looks suspicious.

File: packages/core/models/notices/PollNoticeModel.js [hunk 3]
```diff
@@ -42,7 +40,10 @@ export default class PollNoticeModel extends abstractNoticeModel({
   }
 
   message () {
-    const message = `notices.polls.${this.get('status')}`
-    return I18n.t(message)
+    const value = `notices.polls.${this.get('status')}`
+
+    return {
+      value,
```
Comment (e-maximus) [inline packages/core/models/notices/PollNoticeModel.js]: Can be simplified like
`value:` `notices.polls.${this.get('status')}`

File: packages/core/models/notices/TokenNoticeModel.js [hunk 4]
```diff
@@ -52,10 +50,12 @@ export default class TokenNoticeModel extends abstractNoticeModel({
   }
 
   message () {
-    const message = `notices.settings.erc20.tokens.${this.get('status')}`
-    return I18n.t(message, {
+    const value = `notices.settings.erc20.tokens.${this.get('status')}`
+
+    return {
+      value,
```
Comment (e-maximus) [inline packages/core/models/notices/TokenNoticeModel.js]: Can be simplified like
`value: `notices.settings.erc20.tokens.${this.get('status')}``

File: packages/core/models/notices/TransactionErrorNoticeModel.js [hunk 5]
```diff
@@ -29,13 +28,16 @@ export default class TransactionErrorNoticeModel extends abstractNoticeModel({
   }
 
   message () {
-    const message = `errorCodes.${this.error().code}`
-    return I18n.t(message)
+    const value = `errorCodes.${this.error().code}`
+
+    return {
+      value,
```
Comment (e-maximus) [inline packages/core/models/notices/TransactionErrorNoticeModel.js]: Can be simplified like
`value: `errorCodes.${this.error().code}`

File: packages/core/models/notices/TransferErrorNoticeModel.js [hunk 6]
```diff
@@ -21,22 +20,24 @@ export default class TransferErrorNoticeModel extends abstractNoticeModel({
...
   error (): TransferError {
     return this.get('error')
   }
 
   message () {
-    const message = `notices.transfer.errors.${this.error().code}`
-    return I18n.t(message)
+    const value = `notices.transfer.errors.${this.error().code}`
+    return {
+      value,
```
Comment (e-maximus) [inline packages/core/models/notices/TransferErrorNoticeModel.js]: Can be simplified
`value: `notices.transfer.errors.${this.error().code}``

**Pull request merged**